lib/source/pl/lib/std/pragmas.cpp

```diff
@@ -3,8 +3,24 @@
 #include <pl/core/validator.hpp>
 #include <pl/core/evaluator.hpp>
 
-namespace pl::lib::libstd {
+const long unsigned int LIMITMAX = std::numeric_limits<long unsigned int>::max() - 1;
+
+bool parse_limit(const std::string &value, long unsigned int *limit) {
```

Comment:
Camel case to match the rest of the code.
Also use `u64` and so on for type names.
I'd probably instead return a `std::optional<u64>` here where an empty optional signalizes that an error occurred instead of doing it through an out parameter